content/minfraud/api-documentation/_schemas/ResponseRiskReason.mdx

```diff
@@ -0,0 +1,31 @@
+import responseJson from '../_examples/response';
+
+<MinFraudSchema
+  json={responseJson.risk_score_reasons[0]}
+  jsonPointer="/risk_score_reasons/{n}"
+  name="Response | Risk Score Reason"
+  services={[
+    'factors',
+  ]}
+>
+  This object contains risk score reasons for a given transaction that change the risk score significantly.
```

Comment:
> This object contains risk score reasons

Is it correct for this to be plural? Or should it be e.g. `This object contains a risk score reason`